ligo/src/treasuryDAO.mligo

```diff
@@ -0,0 +1,116 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#include "defaults.mligo"
+#include "types.mligo"
+#include "proposal.mligo"
+#include "helper/unpack.mligo"
+#include "base_DAO.mligo"
+
+#include "treasuryDAO/types.mligo"
+
+// -------------------------------------
+// Helpers
+// -------------------------------------
+
+let unpack_transfer_type_list (b: bytes) : transfer_type list =
+  match ((Bytes.unpack b) : ((transfer_type list) option)) with
+    Some (v) -> v
+  | None -> (failwith "UNPACKING_NOT_NAT_OPTION_TYPE" : transfer_type list)
```

Comment:
I think the message was incorrectly copied from `unpack_nat_option` and is not correct.

Also this function could be moved to `helper/unpack.mligo` I suppose.

Reply:
Yeah, I don't put this in the helper because it is only applicable to treasury dao, whereas `helper` is mean to commonly used function.

Reply:
Ok, I see, it makes sense to leave it as-is in this case :+1: